howtos/kd_integration/images/sys-dev-states.pu

```diff
@@ -0,0 +1,72 @@
+@startuml
+footer: timeline not to scale 
+
+robust "System" as Sys
+Sys has "S0" as high
+Sys has "S0ix" as low
+Sys has "S3" as none
+
+robust "Device" as Dev
+Dev has "D0" as high
+Dev has "D0ix" as low
+Dev has "D3" as none
+
+robust "WOV" as Wov
```

Comment:
Do we want to remove all WOV references and use KD instead ?

Reply:
fixed. I've updated PR to remove all references to files that are in a draft state.